### PR TITLE
Fpm release v0.9.0

### LINKED ISSUE #921: fpm 0.9.0 release
Feature release for Metapackages functionality

### PR DESCRIPTION
**Pull request by perazz**
Closes #921.

Preview release at https://github.com/perazz/fpm/releases/tag/v0.9.0

### CODE REVIEW

File: src/fpm_meta.f90 [hunk 1]
```diff
@@ -725,6 +725,7 @@ logical function msmpi_init(this,compiler,error) result(found)
         !> Add default run command
         this%has_run_command = .true.
         this%run_command = string_t(join_path(get_dos_path(bindir,error),'mpiexec.exe')//' -np * ')
+        print *, this%run_command%s
```
Comment (awvwgk) [inline src/fpm_meta.f90:728]:
```suggestion
```

**Review by awvwgk — CHANGES_REQUESTED** [review]

**Review by awvwgk — CHANGES_REQUESTED** [review]
This introduces a new change for MPI, I would prefer to have last minute feature additions as separate patch.

**Comment by perazz** [comment]
Yes, the MS-MPI CI that seldom does not read %MSMPI_BIN%, and the logs are not helpful. I will stash that commit

**Review by awvwgk — APPROVED** [review]
Looks good to me now

**Comment by awvwgk** [comment]
Thanks for the clarification.

**Pull request merged**